__tests__/store/useTripStore.test.ts

```
@@ -592,34 +611,39 @@ describe('useTripStore', () => {
+            ],
+          },
+        ],
       });
 
       const trip = useTripStore.getState().getTripById('trip-123');
-      expect(trip?.members!.length).toBe(2);
-      const creator = trip?.members!.find(m => m.userId === mockTrip.createdBy);
-      expect(creator).toBeDefined();
-      expect(creator!.role).toBe('owner');
+      // getTripById returns the trip as-is, members are not modified
+      expect(trip?.members!.length).toBe(1);
```

Biome (2.3.13)

[error] Forbidden non-null assertion after optional chaining.

Optional chaining already handles nullish values. Using non-null assertion defeats its purpose and may cause runtime errors.
Consider using the nullish coalescing operator `??` or optional chaining throughout the chain instead.

(lint/suspicious/noNonNullAssertedOptionalChain)